### ISSUE TITLE
Fails to start on RHEL 7.6

### ISSUE THREAD

**Issue by aucampia** [flathub/com.bitwarden.desktop#15]
I get an error when trying to run this flatpak:

```
$ flatpak run com.bitwarden.desktop 
[3:0902/074402.284834:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /app/bin/chrome-sandbox is owned by root and has mode 4755.
```

flatpak info:

```
20190902T074326 E1258170@noosl-0r018:~
$ flatpak info com.bitwarden.desktop/x86_64/stable
Ref: app/com.bitwarden.desktop/x86_64/stable
ID: com.bitwarden.desktop
Arch: x86_64
Branch: stable
Origin: flathub
Collection ID: org.flathub.Stable
Date: 2019-08-15 22:06:04 +0000
Subject: Add category to AppStream metadata (4e30eb73)
Commit: 3d8618e0cdaae053e77ab36bb477a4fb45bdbf045cfed06591d78f5c43469cd5
Parent: ecfcdda55d8139e88a8ee95b1852ffd4534bf2f3767861dc8a97099a9e510eba
Location: /var/lib/flatpak/app/com.bitwarden.desktop/x86_64/stable/3d8618e0cdaae053e77ab36bb477a4fb45bdbf045cfed06591d78f5c43469cd5
Installed size: 177.6 MB
Runtime: org.freedesktop.Platform/x86_64/18.08
Sdk: org.freedesktop.Sdk/x86_64/18.08
```

OS info:

```
20190902T074327 E1258170@noosl-0r018:~
$ cat /etc/os-release
NAME="Red Hat Enterprise Linux Server"
VERSION="7.7 (Maipo)"
ID="rhel"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="7.7"
PRETTY_NAME="Red Hat Enterprise Linux"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:7.7:GA:server"
HOME_URL="https://www.redhat.com/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 7"
REDHAT_BUGZILLA_PRODUCT_VERSION=7.7
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="7.7"
```

**Comment by aucampia** [comment]
Even if I set suid on chrome sandbox (which I don't want to do):

```
root@noosl-0r018:~
# chmod +s /var/lib/flatpak/app/com.bitwarden.desktop/x86_64/stable/3d8618e0cdaae053e77ab36bb477a4fb45bdbf045cfed06591d78f5c43469cd5/files/bin/chrome-sandbox 
root@noosl-0r018:~
# ls -lh /var/lib/flatpak/app/com.bitwarden.desktop/x86_64/stable/3d8618e0cdaae053e77ab36bb477a4fb45bdbf045cfed06591d78f5c43469cd5/files/bin/chrome-sandbox
-rwsr-sr-x. 2 root root 232K Jan  1  1970 /var/lib/flatpak/app/com.bitwarden.desktop/x86_64/stable/3d8618e0cdaae053e77ab36bb477a4fb45bdbf045cfed06591d78f5c43469cd5/files/bin/chrome-sandbox
```

I still get errors:

```
20190902T074625 E1258170@noosl-0r018:~
$ flatpak run com.bitwarden.desktop 
The setuid sandbox is not running as root. Common causes:
  * An unprivileged process using ptrace on it, like a debugger.
  * A parent process set prctl(PR_SET_NO_NEW_PRIVS, ...)
Failed to move to new namespace: PID namespaces supported, Network namespace supported, but failed: errno = Operation not permitted
```

**Comment by ghisvail** [comment]
This issue should be fixed together with #16.

Right now, injection of the necessary `--no-sandbox` option is done in the desktop file, hence why running the app through your desktop launcher of choice works fine.

From the command line however, the option is missing and has to be provided manually : running `flatpak run com.bitwarden.desktop --no-sandbox` should work fine.
